src/modules/github/github.types.js

```diff
@@ -0,0 +1,4 @@
+const SET_REPOS = 'setRepos';
+const SET_ERROR = 'setError';
```

Comment:
Zwróć uwagę, że nazwy akcji są identyczne w przypadku GH oraz SO i dlatego wiadomość o błędzie wpada w oba miejsca. Wystarczy tutaj dodać przyrostek n. GH i problem będzie rozwiązany. To też pokazuje dlaczego warto robić stałą i ją przekazywać dalej. Jeśli wystąpi kolidacja nazw to wystarczy ją zmienić w jednym miejscu :)

PS. Można by było utworzyć też osobny "module" do błędów i problemu by nie było :)